lib/util/createDomain.js

```diff
@@ -6,7 +6,7 @@ const internalIp = require('internal-ip');
 
 module.exports = function createDomain(options, listeningApp) {
   const protocol = options.https ? 'https' : 'http';
-  const appPort = listeningApp ? listeningApp.address().port : 0;
+  const appPort = listeningApp ? listeningApp.address().port : options.port.toString();
```

Comment:
this will break dynamic port assignment, port `0` was intentional there for that reason.